packages/block-library/src/form/deprecated.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * WordPress dependencies
+ */
+import { InnerBlocks, useBlockProps } from '@wordpress/block-editor';
+
+const deprecated = [
+	{
+		// The block supports here are deliberately empty despite this
+		// deprecated version of the block having adopted block supports.
+		// The attributes added by these supports have been manually
+		// added to this deprecated version's attributes definition so
+		// that the data isn't lost on migration. All this is so that the
+		// automatic application of block support classes doesn't occur
+		// as this version of the block had a bug that overrode those
+		// classes. If those block support classes are applied during the
+		// deprecation process, this deprecation doesn't match and won't
+		// run.
+		// @see https://github.com/WordPress/gutenberg/pull/55755
+		supports: {},
+		attributes: {
+			submissionMethod: {
+				type: 'string',
+				default: 'email',
+			},
+			method: {
+				type: 'string',
+				default: 'post',
+			},
+			action: {
+				type: 'string',
+			},
+			email: {
+				type: 'string',
+			},
+			// The following attributes have been added to match the block
+			// supports at the time of the deprecation. See above for details.
```

Comment:
I appreciate the commit adding these is labelled WIP, but just in case it isn't already on the radar, it looks like we still need the `anchor` block supports attribute as added [here](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-editor/src/hooks/anchor.js#L38), given that support was previously adopted.

Reply:
Good  catch! Thanks, added 👍